<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>517</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ServerMessageInitScreen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Logout">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoggedAs">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="layoutwidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>741</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>291</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="NotLoggedLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="Register">
         <property name="text">
          <string>Sign Up</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Login">
         <property name="text">
          <string>Sign In</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="gridStackedWidget_2Page1">
     <layout class="QGridLayout" name="LoggedLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="Transactions">
        <property name="text">
         <string>My Transactions</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="Bids">
        <property name="text">
         <string>My Bids</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="Quotations">
        <property name="text">
         <string>View quotations</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="CreateBid">
        <property name="text">
         <string>Create Bid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="Settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resources/settings-cog.svg</normaloff>
      <normalon>resources/settings-cog.svg</normalon>resources/settings-cog.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>355</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
